.github/workflows/unit-tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         run: ruff . --output-format github
       - name: Unit Tests
         run: |
-          export ACCESS_PRIVATE_DATASETS=1
+          export ACCESS_PRIVATE_REFERENCE_DATASETS=1
```

Comment:
[HGMD](https://www.hgmd.cf.ac.uk/ac/index.php) is a restricted access database that we use to annotate data in seqr, that we actually had to pay for a license to use and the terms of that license do not allow us to broadly share that data. Out main reference data is made available in a completely public google bucket so anyone using a local installation of seqr can use it, but for HGMD we maintain a separate table thats kept in a "private" bucket (i.e. has strict access control) so that we follow the terms of our liscense